tests/console/validate_lvm_raid1.pm

```diff
@@ -14,7 +14,6 @@ use utils;
 use Test::Assert ':all';
 use scheduler 'get_test_suite_data';
 use version_utils "is_sle";
```

Comment:
I think this is not used any more after removal.

Reply:
I think only this one can be removed:
```perl
use version_utils "is_sle";
```

Reply:
In fact, I just mean the line I added comments. :)